apps/code/app.h

```diff
@@ -11,6 +11,8 @@
 #include "variable_box_controller.h"
 #include "../shared/shared_app.h"
 
+#define DEFAULT_PYTHON_HEAP_SIZE 62000
```

Comment:
Why do you decrease the size ?

Reply:
The size from 62000 or from the real default size which is 67000 ? :thinking:

Reply:
In the code before this pull request, the heap size was 67 Ko, but you decreased it to 62 Ko

Reply:
OK